doc/de/NEUES.md

```diff
@@ -61,7 +61,7 @@ Umfangreichere Informationen über die Änderungen des aktuellen Codes findest D
    - Unterstützung für Obsidian hinzugefügt.
    - Unterstützung für Added support for Pink Panther: Passport to Peril hinzugefügt.
    - Unterstützung für Pink Panther: Hokus Pokus Pink hinzugefügt.
-   - Unterstützung für Adibou 2 (Adi Junior) "Environment", "Read/Count 4 & 5" und "Read/Count 6 & 7" hinzugefügt.
+   - Unterstützung für Adibou 2 (Addy Junior) "Environment", "Lesen/Rechnen 4 & 5" und "Lesen/Rechnen 6 & 7" hinzugefügt.
```

Comment:
According to the title screen, we should leave it as "Adi", at least that's how the version I have shows it. I agree with "Lesen/Rechnen" though.
![scummvm-adibou2-win-de-00001](https://user-images.githubusercontent.com/11882577/223514470-33e9d66b-0d7a-4fd6-91ff-63f6a7ffd253.png)